app/models/OtherExpense.scala

```diff
@@ -18,8 +18,8 @@ package models
 
```

Comment:
Better to name this file `IABDExpense.scala`

Reply:
Pushed a commit with a rename of the file.